projects/alexlab/index.js

```diff
@@ -1,44 +1,21 @@
-const sdk = require("@defillama/sdk");
-const BigNumber = require("bignumber.js");
 const axios = require("axios");
-const retry = require('../helper/retry');
+const retry = require("../helper/retry");
 
 // https://explorer.stacks.co/txid/SP3K8BC0PPEVCV7NZ6QSRWPQ2JE9E5B6N3PA0KBR9.alex-vault?chain=mainnet
 // https://stacks-node-api.blockstack.org/extended/v1/address/SP3K8BC0PPEVCV7NZ6QSRWPQ2JE9E5B6N3PA0KBR9.alex-vault/balances
-const alex_vault = 'SP3K8BC0PPEVCV7NZ6QSRWPQ2JE9E5B6N3PA0KBR9.alex-vault';
-const STACKS_API = 'https://stacks-node-api.mainnet.stacks.co/extended/v1/address'
+const ALEX_API = "https://api.alexlab.co/v1";
 
-const transformAddress = addr => {
-  const slug = addr.substr(addr.indexOf('::') + 2);
-  return slug
+async function fetch() {
+  const url = `${ALEX_API}/stats/tvl`;
+  const alexResponse = (await retry(async () => await axios.get(url))).data;
+  return alexResponse.lp_token_supply;
```

Comment:
should change to alexResponse.tvl

Reply:
wouldnt that be double counting the LPs though (once in the AMM, and again when staked)

Reply:
at defillama we only count the inital deposit to the LP pool and dont count further staking of the LPs. We use this method for all protocols